src/loader.js

```diff
@@ -44,7 +44,7 @@ export function loadGooglePlacesLibrary(apiKey, callback) {
   element.async = true
   element.defer = true
   element.onload = onLibraryLoaded
-  element.src = `https://maps.googleapis.com/maps/api/js?key=${encodeURIComponent(apiKey)}&libraries=places`
+  element.src = `https://maps.googleapis.com/maps/api/js?key=${encodeURIComponent(apiKey)}&libraries=places&loading=async&callback=Function.prototype`
```

Comment:
Unfortunately, I think we need take out the `&loading=async` since we _are_ using the script tag's "load" event. Per [the docs](https://developers.google.com/maps/documentation/javascript/load-maps-js-api#optional_parameters_direct):

> `loading`: The code loading strategy that the Maps JavaScript API can use. Set to async to indicate that the Maps JavaScript API has not been loaded synchronously and that no JavaScript code is triggered by the script's load event. It is highly recommended to set this to async whenever possible, for improved performance. (Use the callback parameter instead to perform actions when the Maps JavaScript API is available.) Available starting with version 3.55.

If we were able to find a simple/reliable way to use the `callback` instead of `onload`, then setting `loading=async` sounds like it would be great. However, that should probably be a separate PR at this point (if it's even needed at the moment).
```suggestion
  element.src = `https://maps.googleapis.com/maps/api/js?key=${encodeURIComponent(apiKey)}&libraries=places&callback=Function.prototype`
```

Reply:
I'm going to go ahead and commit this change and merge this PR, so that the original problem will be fixed.

@cptcrunchy Thank you so much for helping us with this. Feel free to open another PR to try to get the `loading=async` working, if desired.